src/types.ts

```diff
@@ -205,6 +205,7 @@ export type Config = {
   AUTH0_ALLOW_DELETE: boolean;
   AUTH0_EXCLUDED?: AssetTypes[];
   AUTH0_INCLUDED_ONLY?: AssetTypes[];
+  AUTH0_PRESERVE_KEYWORDS: boolean;
```

Comment:
Introduction of the configuration to enable user opt-in behavior. Will be integrated as a flag in a later PR.

Reply:
💡 🤔 What if we keep it as a config key only?